test/misc.jl

```diff
@@ -1363,9 +1363,9 @@ end
 @test isdefined(KwdefWithEsc_TestModule, :Struct)
 
 @testset "exports of modules" begin
-    for (_, mod) in Base.loaded_modules
+    @testset "$mod" for (_, mod) in Base.loaded_modules
```

Comment:
Are these nested correctly? It seems like `mod` is defined in the inner scope

Reply:
Yeah, that's how `@testset for` can be set up. Frustratingly the testset report prints in `Base.runtests` don't expand deeper than one level if they fail, so this doesn't help identify what's failed here.

Reply:
Oh wow, I had no idea - Good to know